modules/programs/timidity.nix

```diff
@@ -0,0 +1,48 @@
+{ config, lib, pkgs, ... }:
+
+let cfg = config.programs.timidity;
+in {
+  meta.maintainers = [ lib.maintainers.amesgen ];
+
+  options.programs.timidity = {
+    enable = lib.mkEnableOption "timidity, a software MIDI renderer";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.timidity;
+      defaultText = lib.literalExpression "pkgs.timidity";
+      description = "The timidity package to use.";
+    };
+
+    wrappedPackage = lib.mkOption {
+      readOnly = true;
+      type = lib.types.package;
+      description = "Resulting package.";
+    };
+
+    extraConfig = lib.mkOption {
+      type = lib.types.lines;
+      default = "";
+      example = ''
+        soundfont ''${pkgs.soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2
+      '';
```

Comment:
```suggestion
      example = lib.literalExpession '''''
        soundfont ''${pkgs.soundfont-fluid}/share/soundfonts/FluidR3_GM2-2.sf2
      ''''';
```

Reply:
Thanks :+1:, I applied it manually due to `literalExpession` vs `literalExpression`.